src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -154,6 +149,63 @@ docker-compose --verbose up --exit-code-from helloworld
 If splitting by space is prohibitive you can specify the split regex in the <<_glossary_generic_information, generic informations>> with the
 key *docker-compose-options-split-regex*. If you supply e.g. "!SPLIT!" as value, then your *docker-compose-up-options* will need to look like this: "--option1!SPLIT!--option2".
 
+=== Docker_File task
+
+The basic behavior of a Docker_File task is create an image with Docker build and create a container based on the built image. At the end of the job, the created image and container are deleted. And, advanced options will allow to do more specific actions.
+
+In order for Docker_File tasks to work, the _Node_ must have Docker installed, please refer to the official
+ Docker documentation to see how to install http://docs.docker.com/engine/installation/[Docker^]
+
+A Docker_File task expects the content of a Dockerfile inside the _Script_ section. You can find out how to write Dockerfile
+with the official https://docs.docker.com/engine/reference/builder/[Dockerfile documentation^].
+
+To get started: a simple Docker_File task example is explained below.
+
+The content of the _Script_ section (equal to the content of a Dockerfile) is:
+----
+FROM ubuntu:18.04
+RUN echo "Hello ProActive »
+RUN sleep 30
+----
+The above example is a simple task that will create an image with Docker build.
+After that a container, which will run the commands 'echo "Hello ProActive"' and 'sleep 30', is created from the built image.
+
+
+* The Docker_File task allows to set parameters to the `docker-build` command with regard to the docker-build CLI https://docs.docker.com/engine/reference/builder/[reference].
+----
+docker build [OPTIONS] PATH | URL | -
+----
+
+To define a _docker-build option_, use the generic information *docker-build-options* 
```

Comment:
make a cross reference to the generic information section in the documentation.

Rather than pointing to the glossary (like above : <<_glossary_generic_information, generic informations>>), it is better to point to the real generic information chapter.